src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -81,8 +80,9 @@ const RadioFrequencyLink = styled(Link)`
   ${({ service }) => service && getSansRegular(service)};
 `;
 
-const CanonicalRadioSchedule = ({ endpoint }) => {
-  const [schedule, setRadioSchedule] = useState();
+const CanonicalRadioSchedule = ({ endpoint, initialData }) => {
+  const radioScheduleData = useData(endpoint, initialData);
```

Comment:
I was wondering what the use case for having `useData` is for radio schedules? Is it just to protect us from the chance we might not get data from the server?

Reply:
So with this PR I took what we already had around client side data fetching in the component, and then tried to add initial data in from the server. The simplest thing we could do is actually something like this: #6094 (branched off this PR), where we just pass down the data into the radio schedule as prop. Something we can consider on Monday. What's nice about a pattern like the `useData` pattern above is that we can just place the component somewhere and allow it to fetch the data it needs by itself - but it might be worth adding that complexity in once we need it.